101-XSS-Attack-Prevention/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "XSS attack on webapp Scenario",
+    "description": "This will deploy 2 application gateway, 1 webapp, sql server and database, OMS and other network resources. One app gateway is on detection mode and other is on prevention mode. Perform the XXS attack by following the guidlines and execute the scenario for mitigation and prevention of a XXS attack.",
+    "summary": "Showcase a XXS attack on webapp - detection & prevention.",
```

Comment:
Showcase detection and prevention of a XSS attack on an Azure Weg App

Reply:
updated as per recommended